etl-scripts/stored-procedures/generate_flat_hiv_summary.sql

```diff
@@ -514,12 +514,18 @@ SELECT @person_ids_count AS 'num patients to sync';
                              ELSE @enrollment_location_id:=NULL
                          END AS enrollment_location_id,
 
-                         null as ovc_non_enrollment_reason,
-                         null as ovc_non_enrollment_date,
-
-                            
-                            
-                            
+                         case 
+                            when obs regexp "!!11219=6834" then @ovc_non_enrollment_reason := 1
```

Comment:
I think using 6834 and 1504 instead of 1 and 2 respectively, is more appropriate, as it prevent recoding of these concepts. One will need to come back to these scripts to try and figure out what 1 and 2 mean. Some of the consumers might not be so technical (M & E).